packages/core/src/components/Button.js

```diff
@@ -1,82 +1,63 @@
 // @flow
 
 import React, { Component } from 'react'
-import { TouchableOpacity, Text, StyleSheet } from 'react-native-web'
-import COLORS from '../colors'
+import { Text } from 'react-native'
+import { css } from 'styled-components/native'
 
-type Props = {
-  title: string,
+import HoverView from './HoverView'
+
+type ContainerProps = {
   disabled?: boolean,
-  buttonStyle?: number | Array<number | Object> | Object,
-  textStyle?: number | Array<number | Object> | Object,
-  outlineStyle?: boolean,
+  outlined?: boolean,
   onPress?: Function,
 }
 
-export default class Button extends Component<Props> {
-  render() {
-    const {
-      title,
-      disabled,
-      outlineStyle,
-      buttonStyle,
-      textStyle,
-      onPress,
-    } = this.props
-
-    const buttonStyles = [styles.button]
-    const textStyles = [styles.text]
-
-    if (disabled) {
-      buttonStyles.push(styles.disabled)
-    }
-
-    if (outlineStyle) {
-      buttonStyles.push(styles.outline)
-      textStyles.push(styles.redText)
-    }
+type Props = ContainerProps & {
+  children: any,
```

Comment:
Ideally the `any` type should never be used, in this case it should have the `Node` type from the `react` package, see https://flow.org/en/docs/react/children/